### PR TITLE
Brand - new backgroundColour & logoColour props

### PR DESCRIPTION
**Pull request by sareh**
No ticket
A new brand colour will be needed shortly: https://github.com/bbc/psammead/pull/2120

We could build a theming object and pass that in, but in the case of Brand it's simpler to pass in just these two props instead. 

**Overall change:** Update brand to take in colour props `backgroundColour` & `logoColour`. These are required props since there is no default colour, so this is a major version increase. 

**Code changes:**

- Update brand to take in colour props backgroundColour & logoColour
- Add color storybook addon knob to change backgroundColour & logoColour

Storybook with standard colours:
Screenshot of brand component in Storybook

Can change the colours using the Storybook color add-on knob
Screenshot of brand component in Storybook, with backgroundColour changed

Localhost Storybook link: http://localhost:8180/?path=/story/components-brand--with-brand-link

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-brand/README.md [hunk 1]
```diff
@@ -57,6 +62,8 @@ const Header = (product, serviceName) => (
       minWidth={180}
       svg={igbo}
       url="https://www.bbc.co.uk/news"
+      backgroundColour={backgroundColour}
+      logoColour={logoColour}
```
Comment (DenisHdz) [inline packages/components/psammead-brand/README.md:66]: LGTM, but what about adding the `color` addon so we can also change the colors from the Storybook?

```
import {  color } from '@storybook/addon-knobs';
const backgroundColour = color('POSTBOX', `${C_POSTBOX}`);
const logoColour = color('WHITE', `${C_WHITE}`);
```

color-addon

Comment (sareh) [reply packages/components/psammead-brand/README.md:66]: Thanks. Have updated it. I used the labels `Background colour` and `Logo colour` to keep them in keeping with the other knob labels that refer to the prop name.

**Review by DenisHdz — APPROVED** [review]
LGTM :)

**Review by EinsteinNjoroge — APPROVED** [review]
LGTM

**Review by jroebu14 — APPROVED** [review]

**Comment by staylos92** [comment]
Happy for this to be merged :+1:

**Pull request merged**